src/components/map/Popups/StudyPopup.tsx

```diff
@@ -52,30 +52,41 @@ function getTranslatedPopulationGroup(popGroupOptions: Record<string, string>[],
+                    {record.url ? <a href={record.url} target="_blank" rel="noopener noreferrer">{record.source_name}</a> : record.source_name}
+                    {/*{getGeography(record.city, record.state, record.country)}*/}
                 </div>
             </div>
-            <Divider />
-            {row(Translate("StudyName"), record.url ? <a href={record.url} target="_blank" rel="noopener noreferrer">{record.source_name}</a> : record.source_name)}
-            {/*{row("Scope", record.estimate_grade)}*/}
-            {/*{row(Translate(`${record.estimate_grade}StudyDetails`), getGeography(record.city, record.state, record.country))}*/}
-            {(record.sampling_start_date && record.sampling_end_date) && (
-                <>
-                    {row(Translate("SamplingDates"), `${TranslateDate(record.sampling_start_date)} → ${TranslateDate(record.sampling_end_date)}`)}
-                </>)
-            }
-            {row(Translate("PopulationGroup"), record.population_group ? getTranslatedPopulationGroup(popGroupOptions, record.population_group) : Translate("NotReported"))}
-            {row(Translate("BestSeroprevalenceEstimate"),record.serum_pos_prevalence ? `${(record.serum_pos_prevalence * 100).toFixed(1)}%` : "N/A")}
-            {row(Translate("SampleSize"), record.denominator_value?.toString().replace(/\B(?=(\d{3})+(?!\d))/g, Translate(",")))}
-            {row(Translate("AntibodyTarget"), record.antibody_target && record.antibody_target.length > 0 ? (record.antibody_target.length === 2 ? record.antibody_target.join(", ") : record.antibody_target) : "N/A")}
-            {row(Translate("PositiveCases"), (record.cases_per_hundred ? record.cases_per_hundred.toFixed(1) + " per 100" : "N/A"))}
-            {row(Translate("Vaccinations"), (record.full_vaccinations_per_hundred ? record.full_vaccinations_per_hundred.toFixed(1) + " per 100" : "N/A"))}
+            {/*SeroPrev section*/}
+            <div className={"popup-seroprev-level popup-section"}>
+                <div className={"popup-heading"}>
+                    {Translate("BestSeroprevalenceEstimate")}: <b> {record.serum_pos_prevalence ? `${(record.serum_pos_prevalence * 100).toFixed(1)}%` : "N/A"}</b>
+                </div>
+                <div className={"popup-text"}>
+                    {console.log((record.sampling_start_date?.substring(0, 10).replace(/-/g, '/')))}
+                    {(record.sampling_start_date && record.sampling_end_date) && (
+                        <>
+                            {`${NumberDateToWordDate(record.sampling_start_date.substring(0, 10).replace(/-/g, '/'))} → ${NumberDateToWordDate(record.sampling_end_date.substring(0, 10).replace(/-/g, '/'))}`}
+                        </>)
+                    }
+                </div>
+            </div>
+            {/*Content section*/}
+            <div className={"popup-section"}>
+                {row(Translate("PopulationGroup"), record.population_group ?? Translate("NotReported"))}
```

Comment:
```suggestion
                            {row(Translate("PopulationGroup"), record.population_group ? getTranslatedPopulationGroup(popGroupOptions, record.population_group) : Translate("NotReported"))}

```

Reply:
Need this to ensure population groups are translated properly, as their translations are dynamic (come from API response) rather than defined in json files